perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupsManagerEntryIntegrationTest.java

```diff
@@ -5324,6 +5318,53 @@ public void getGroupRichMembersByIds_containsGroupStatuses() throws Exception {
 		assertFalse(richMembers.get(0).getGroupStatuses().isEmpty());
 	}
 
+	@Test
```

Comment:
I think you can include one more test where you would test some use-case with subgroups. Somethink like this:

```
  A
  |
  B
```
Where a member would be expired in the group `A` and valid in the group `B`.

Reply:
Added.